### PR TITLE
fix a couple of bugs, add Tag tests

### PR DESCRIPTION
**Pull request by jonsequitur**

### CODE REVIEW

File: Microsoft.DotNet.Interactive.Rendering.Tests/HtmlFormatterTests.cs [hunk 1]
```diff
@@ -241,6 +241,18 @@ public void It_formats_dictionaries_as_tables_with_the_key_on_the_y_axis()
                       .Be(
                           "<table><thead><tr><th><i>key</i></th><th>TypeName</th><th>Id</th></tr></thead><tbody><tr><td>first</td><td>entity one</td><td>123</td></tr><tr><td>second</td><td>entity two</td><td>456</td></tr></tbody></table>");
             }
+
+            [Fact]
+            public void It_formats_arrays_of_disparate_types_good()
```
Comment (colombod) [inline Microsoft.DotNet.Interactive.Rendering.Tests/HtmlFormatterTests.cs]: will this make PR builds fail?

File: rendering.sln [hunk 2]
```diff
@@ -0,0 +1,48 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (colombod) [inline rendering.sln]: why this solution?

File: Microsoft.DotNet.Interactive.Rendering.Tests/HtmlFormatterTests.cs (hunk 1 above)
Comment (jonsequitur) [reply Microsoft.DotNet.Interactive.Rendering.Tests/HtmlFormatterTests.cs]: Ooops, meant to exclude that.

File: rendering.sln (hunk 2 above)
Comment (jonsequitur) [reply rendering.sln]: Meant to exclude this.

**Pull request merged**